.github/workflows/build.yml

```diff
@@ -80,6 +80,15 @@ jobs:
           sudo dpkg -i intel-igc-core_1.0.12812.24_amd64.deb intel-level-zero-gpu_1.3.25018.24_amd64.deb libigdgmm12_22.3.0_amd64.deb intel-igc-opencl_1.0.12812.24_amd64.deb
           conda install -n omnisci-dev -c conda-forge level-zero-devel pkg-config
 
+      - name: Save Conda env cache
+        if: steps.conda-cache.outputs.cache-hit != 'true'
```

Comment:
better to name ids for the same steps the same way imho; use either conda-cache or restore-conda-cache, not both in different places

Reply:
done